src/Umbraco.Web.UI/config/serilog.Release.config

```diff
@@ -17,13 +17,16 @@
 
         <!-- Default JSON log file -->
         <!-- This is used by the default log viewer in the Umbraco backoffice -->
-        <add key="serilog:using:File" value="Serilog.Sinks.File" />
+        <add key="serilog:using:File" value="Umbraco.Core" />
```

Comment:
@bergmania With these config changes we're going to need a xdt transform so that both Cloud and Nuget upgrades get the changes auto applied. I'll get one made.

Reply:
@bergmania don't we also want to apply these changes to the non-release config file?

Reply:
There don't exists a non-release one. That file is copied from the release one in a before build step in `Umbraco.Web.UI.cspoj`. I don't know the reason for this.

```
<!-- Create Serilog.config & serilog.user.config file from Templates if it doesn't exist -->
    <Message Text="Copy serilog.$(Configuration).config to serilog.config" Importance="high" Condition="!Exists('$(ProjectDir)Config\serilog.config')" />
    <Copy SourceFiles="$(ProjectDir)Config\serilog.Release.config" DestinationFiles="$(ProjectDir)Config\serilog.config" OverwriteReadOnlyFiles="true" SkipUnchangedFiles="false" Condition="!Exists('$(ProjectDir)Config\serilog.config')" />
    <Message Text="Copy serilog.user.$(Configuration).config to serilog.user.config" Importance="high" Condition="!Exists('$(ProjectDir)Config\serilog.user.config')" />
    <Copy SourceFiles="$(ProjectDir)Config\serilog.user.Release.config" DestinationFiles="$(ProjectDir)Config\serilog.user.config" OverwriteReadOnlyFiles="true" SkipUnchangedFiles="false" Condition="!Exists('$(ProjectDir)Config\serilog.user.config')" />
    
```

Reply:
ah of course 🤦‍♂ it's just for developer testing, that's the only reason it's not checked in, not sure when/why that decision was made but that's how it is i guess since that will be the same for some other config files.